src/django_components/component.py

```diff
@@ -110,6 +111,38 @@ class ViewFn(Protocol):
     def __call__(self, request: HttpRequest, *args: Any, **kwargs: Any) -> Any: ...  # noqa: E704
 
 
+class ComponentVars(NamedTuple):
```

Comment:
In the script that will generate the API reference, I will be importing this class and use its docstrings to document the variables available in the django templates.